systemd/Makefile

```diff
@@ -1,13 +1,27 @@
 .PHONY: install uninstall
 
+PREFIX = /usr/local
+SYSTEMD_DIR = /usr/lib/systemd
+SYSTEMD_UNIT_DIR = $(SYSTEMD_DIR)/system
+SYSTEMD_USER_DIR = $(SYSTEMD_DIR)/user
+
 install:
-	install -Dm0644 popub-local@.service /usr/lib/systemd/system/popub-local@.service
-	install -Dm0644 popub-relay@.service /usr/lib/systemd/system/popub-relay@.service
-	install -Dm0644 example-local.conf /etc/popub/local/example.conf
-	install -Dm0644 example-relay.conf /etc/popub/relay/example.conf
+	sed "s|<PREFIX>|$(PREFIX)|; s|<USER>|nobody|" popub-local@.service.in > popub-local@.service
+	sed "s|<PREFIX>|$(PREFIX)|; s|<USER>|nobody|" popub-relay@.service.in > popub-relay@.service
+	install -Dm0644 popub-local@.service $(DESTDIR)$(SYSTEMD_UNIT_DIR)/popub-local@.service
+	install -Dm0644 popub-relay@.service $(DESTDIR)$(SYSTEMD_UNIT_DIR)/popub-relay@.service
+
+	sed "s|<PREFIX>|$(PREFIX)|; s|<USER>|%u|" popub-local@.service.in > popub-local@.service
+	sed "s|<PREFIX>|$(PREFIX)|; s|<USER>|%u|" popub-relay@.service.in > popub-relay@.service
+	install -Dm0644 popub-local@.service $(DESTDIR)$(SYSTEMD_USER_DIR)/popub-local@.service
+	install -Dm0644 popub-relay@.service $(DESTDIR)$(SYSTEMD_USER_DIR)/popub-relay@.service
+
+	install -Dm0644 example-local.conf $(DESTDIR)/etc/popub/local.d/example
+	install -Dm0644 example-relay.conf $(DESTDIR)/etc/popub/relay.d/example
 	systemctl daemon-reload || true
 
 uninstall:
-	rm -f /usr/lib/systemd/system/popub-local@.service /usr/lib/systemd/system/popub-relay@.service
-	rm -rf /etc/popub/
+	rm -f $(DESTDIR)$(SYSTEMD_UNIT_DIR)/popub-{local,relay}@.service
+	rm -f $(DESTDIR)$(SYSTEMD_USER_DIR)/popub-{local,relay}@.service
+	rm -rf $(DESTDIR)/etc/popub/
```

Comment:
Although most build services guarantee no space in `$DESTDIR`, would you please add quotes to ensure nothing will break if it contains space (or any other characters)?

Thanks.